lib/hooks/useList.ts

```diff
@@ -118,4 +171,32 @@ const useList = (listId: string) => {
   };
 };
 
-export { useList, useFetchAllLists, useFetchListContributors, addListContributor };
+export type TaggedRepoObject = { full_name: string };
+interface PaginatedHighlightsRepoProps {
+  data: TaggedRepoObject[];
+  meta: Meta;
+}
+
+const useListHighlightsTaggedRepos = (listId: string) => {
+  const { data, error, mutate } = useSWR<PaginatedHighlightsRepoProps, Error>(
+    `lists/${listId}/contributors/highlights/tagged-repos?limit=50&page=1`,
```

Comment:
Any reason `limit=50` is hard-coded?

Reply:
We don't have any UI provisions to update this limit programatically ATM, so i assumed setting to a limit of 50 will be a goo start for now.

Reply:
We don't have any UI provisions to update this limit programatically ATM, so i assumed setting to a limit of 50 will be a goo start for now.

Reply:
Cool cool. We can iterate on it.